src/views/tickets/view-link.php

```diff
@@ -55,13 +56,13 @@
 	return false;
 }
 
-$link = $view->get_tickets_page_url( $event_id, $is_event_page );
+$link = $view->get_tickets_page_url( $post_id, $is_event_page );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $link

<sub>hash: 064214d8d15658111af8ebae4fadea21</sub>